apps/user/src/user.service.ts

```diff
@@ -296,6 +296,13 @@ export class UserService {
         throw new NotFoundException(ResponseMessages.user.error.emailNotVerified);
       }
 
+      const decryptedPassword = await this.commonService.decryptPassword(userInfo.password);
```

Comment:
should we generate the hash of the password and send it to the keycloak?

Reply:
Right now we are directly passing the password to the keycloak as it is.